src/sql/src/statements/insert.rs

```diff
@@ -146,4 +146,22 @@ mod tests {
             _ => unreachable!(),
         }
     }
+
+    #[test]
+    fn test_insert_value_with_default() {
+        use crate::statements::statement::Statement;
+
+        // insert "default"
+        let sql = "INSERT INTO my_table VALUES(default)";
```

Comment:
If the `default` is in uppercase, the parsed result is `default` in lowercase too? Maybe we should test it.

Reply:
Had a simple test, the parsed result stays `DEFAULT`, will update the code and unit test later.

Nice catch.